core/spec/models/spree/calculator/refunds/default_refund_amount_spec.rb

```diff
@@ -38,14 +55,44 @@
     end
   end
 
-  context "an exchange" do
+  context 'an exchange' do
     let(:return_item) { build(:exchange_return_item) }
 
     it { is_expected.to eq 0.0 }
   end
 
-  context "pre_tax_amount is zero" do
-    let(:pre_tax_amount)  { 0.0 }
+  context 'pre_tax_amount is zero' do
+    let(:line_item_price)  { 0.0 }
     it { should eq 0.0 }
   end
 end
+
+describe Spree::Calculator::Returns::DefaultRefundAmount, type: :model do
+  context 'tax included in price' do
+    let(:tax_included_in_price) { true }
+    let(:pre_tax_amount) { line_item.pre_tax_amount }
+
+    let(:line_item) { create(:line_item, price: line_item_price, quantity: line_item_quantity) }
+
+    let(:expected_full_refund_amount) do
+      (calculator.compute(return_item) * line_item_quantity) * (1 + tax_amount)
+    end
+
+    it_should_behave_like 'a default refund calculator'
+  end
+
+  context 'tax additional to price' do
+    let(:tax_included_in_price) { false }
+    let(:pre_tax_amount) { line_item_price * line_item_quantity }
+
+    let(:line_item) { create(:line_item, price: line_item_price,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.